### PR TITLE
Add basic test for bootstrap.

### LINKED ISSUE #36: figure out how to test bootstrap command
We should figure out how to test the _bootstrap_ command. Like _offline_ (#22), its effects are all local, so this should be straightforward.

### PR DESCRIPTION
**Pull request by brendandahl**
I had some issues with the gulp-install not triggering a finish event, but that can be fixed with calling sink() at the end. However, I have install disabled for tests as it is slow and require network connectivity.

Fixes #36

### CODE REVIEW

**Comment by marco-c** [comment]
Looks good to me, I have a single question. Why do you need the _noInstall_ option? Does the test fail on Travis if you try to install the dependencies? Or did you add it because otherwise the test takes too much time?

File: lib/bootstrap.js [hunk 1]
```diff
@@ -131,7 +154,8 @@ module.exports = function(config) {
           .pipe(template(templateConfig))
           .pipe(conflict(rootDir))
           .pipe(gulp.dest(rootDir))
-          .pipe(install());
+          .pipe(config.noInstall ? util.noop() : install())
```
Comment (marco-c) [inline lib/bootstrap.js]: Nit: couldn't you directly use sink here instead of util.noop and then sink?

Comment (brendandahl) [inline lib/bootstrap.js]: Install also doesn't work if you don't do a sink().

**Comment by brendandahl** [comment]
> Why do you need the noInstall option?

Mainly because it's slow as mentioned above. Let me try on travis and see if how it's doing.

File: lib/bootstrap.js (hunk 1 above)
Comment (marco-c) [inline lib/bootstrap.js]: Ah, ok. Could you update the comment in the following line then?

**Comment by marco-c** [comment]
> Mainly because it's slow as mentioned above

Oops, I've skimmed over the first comment! I really didn't notice you wrote something in it :smile:

**Comment by marco-c** [comment]
There are a few _npm ERR! tar.unpack untar error_ and then the test times out.

**Pull request merged**